config/mediaflux.yml

```diff
@@ -53,7 +53,7 @@ test:
   api_root_collection_namespace: '/td-test-001'
   api_root_collection: 'path=/td-test-001/tigerdata'
   api_transport: 'http'
-  api_host: 'mediaflux.example.com'
+  api_host: '0.0.0.0'
```

Comment:
Darn...this breaks a ton of tests. Need to figure out how to un-stub only certain calls to Mediaflux.

Reply:
@hectorcorrea 'connect_to_mediaflux: true' on any tests un stubs just that one